packages/amplify_core/test/datstore/test/testData/Person.dart

```diff
@@ -13,40 +13,32 @@
 * permissions and limitations under the License.
 */
 
-// ignore_for_file: public_member_api_docs
+// NOTE: This file is generated and may not follow lint rules defined in your app
+// Generated files can be excluded from analysis in analysis_options.yaml
+// For more info, see: https://dart.dev/guides/language/analysis-options#excluding-code-from-analysis
+
+// ignore_for_file: public_member_api_docs, file_names, unnecessary_new, prefer_if_null_operators, prefer_const_constructors, slash_for_doc_comments, annotate_overrides, non_constant_identifier_names, unnecessary_string_interpolations, prefer_adjacent_string_concatenation, unnecessary_const, dead_code, implicit_dynamic_map_literal, implicit_dynamic_parameter, implicit_dynamic_type
 
 import 'ModelProvider.dart';
-import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+import 'package:amplify_core/amplify_core.dart';
```

Comment:
Oh I guess that change has already been made in codegen. Perfect! Let's ditch the plugin interface

Reply:
I would say we should keep it for the time being, otherwise some people might be forced to regenerate their models, right?

Reply:
It will still be published on pub.dev. If they update Amplify deps and have not regenerated their models in a while, then yes, they'll be required to regenerate them.